python/cudf/cudf/tests/test_parquet.py

```diff
@@ -2545,3 +2545,21 @@ def test_parquet_columns_and_index_param(index, columns):
     got = cudf.read_parquet(buffer, columns=columns)
 
     assert_eq(expected, got, check_index_type=True)
+
+
+def test_parquet_writer_zstd():
+    size = 12345
+    expected = cudf.DataFrame(
+        {
+            "a": np.arange(0, stop=size, dtype="int64"),
```

Comment:
Might be nice to have this test `float64` instead since column `"c"` is `int64` too.